### DISCUSSION TITLE
Idea: Yearly evolution

### DISCUSSION THREAD

**Discussion by corradio** [electricitymaps/electricitymaps-contrib#2114] [Ideas] [answered]
We could build yearly.electricitymap.org which could show yearly averages data.
In the history graphs, we could show the yearly evolution
Thumbs up if you like this idea

**Comment by GilbertKouame** [comment]
ok for it !

**Comment by thomasgibon** [comment]
The IEA has monthly data for OECD countries and a bit more here https://www.oecd.ilibrary.org/energy/data/iea-electricity-information-statistics_elect-data-en
![image](https://user-images.githubusercontent.com/23332955/70137032-65107a80-168d-11ea-9d7f-377f2eab3871.png)

**Comment by scriptator** [comment]
> We could build yearly.electricitymap.org which could show yearly averages data.
> In the history graphs, we could show the yearly evolution

Why not also support monthly? Should not make much of a difference to implement and it would be super interesting to see inter-year fluctuations.

**Comment by corradio** [comment]
you're right - it's just that for some areas we only have yearly data. I'd
be sad to ignore some areas because we lack some monthly data.
Another way to do it is to have a toggle where you can select between
yearly, monthly, and hourly (what is currently used) data.

On Wed, Dec 4, 2019 at 5:32 PM Johannes Vass <notifications@github.com>
wrote:

> We could build yearly.electricitymap.org which could show yearly averages
> data.
> In the history graphs, we could show the yearly evolution
>
> Why not also support monthly? Should not make much of a difference to
> implement and it would be super interesting to see inter-year fluctuations.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tmrowco/electricitymap-contrib/issues/2114?email_source=notifications&email_token=AAMUIKCGOCFBZK2CM6PWL7DQW7LQTA5CNFSM4JVGG2XKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEF5TXLA#issuecomment-561724332>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAMUIKHLVSQXI3CCPG3SAZLQW7LQTANCNFSM4JVGG2XA>
> .
>


-- 

Olivier Corradi

Founder, CEO

<https://www.linkedin.com/in/oliviercorradi> <https://twitter.com/corradio>

tmrow.com

**Comment by brandongalbraith** [comment]
I really :heart: this idea, as it gives you a sense of decarbonization velocity for each measured region.

**Comment by thomasgibon** [comment]
Yearly only sounds great, because it will cover as many countries as possible. For the countries that also have the hourly/15-minute data, you can even validate (/invalidate?) the IEA data :)

**Comment by chiefymuc** [comment]
I always thought a slider between "live", "yearly", "monthly", "daily" and "best available" would be awesome. That way the map looks much more complete. Of course it should be indicated what country uses which "best available" data not to confuse anyone.

Somewhat similar is how the flightradar24 website works. When they lose the actual data of a flight over the atlantic or pacific ocean, they still display the airplane with its flight number and it says "position estimated".

**Comment by robertahunt** [comment]
The BP statistical review provides annual TWh production by fuel for 72 countries going all the way back to 1985 (It seems they cover the whole world, but they aggregate many smaller countries together). https://www.bp.com/content/dam/bp/business-sites/en/global/corporate/xlsx/energy-economics/statistical-review/bp-stats-review-2019-all-data.xlsx

**Comment by VIKTORVAV99** [comment, marked as answer]
This is now available with the recent release of the Historical View when the available data is sufficient.
